requirements.txt

```diff
@@ -1,4 +1,4 @@
 colorlog==6.9.0
-homeassistant==2024.11.0
```

Comment:
<https://github.com/ludeeus/integration_blueprint/blob/main/hacs.json#L3> please update this as well